tests/test_array/good/aggregation_2.catala_en

```diff
@@ -29,13 +29,13 @@ scope B:
 $ catala Interpret -s A
 [RESULT] Computation successful! Results:
 [RESULT] x =
-           [S {"id"= 0; "income"= $0.00}; S {"id"= 1; "income"= $9.00};
-             S {"id"= 2; "income"= $5.20}]
+           [ S { "id"= 0; "income"= $0.00 };  S { "id"= 1; "income"= $9.00 };
```

Comment:
Why the extra space here ?

Reply:
especially before the struct name

Reply:
Good remark, that's suspicious indeed, I'll see how the printer got changed.